// Copyright (c) 2020 Box Bot Inc.
// All rights reserved.

syntax = "proto3";

package testserver;

service Api {

  rpc DoThing (DoThingRequest) returns (DoThingResponse) {}
}

message DoThingRequest {
 string message = 1;
}

message DoThingResponse {
 string message = 1;
}
